apps/dotcom/client/playwright.config.ts

```diff
@@ -0,0 +1,89 @@
+import { defineConfig, devices } from '@playwright/test'
+import path from 'path'
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// import dotenv from 'dotenv';
+// import path from 'path';
+// dotenv.config({ path: path.resolve(__dirname, '.env') });
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+	testDir: './e2e',
+	/* Run tests in files in parallel */
+	fullyParallel: true,
+	/* Fail the build on CI if you accidentally left test.only in the source code. */
+	forbidOnly: !!process.env.CI,
+	/* Retry on CI only */
+	retries: process.env.CI ? 2 : 0,
+	/* Opt out of parallel tests on CI. */
+	workers: process.env.CI ? 1 : undefined,
+	/* Reporter to use. See https://playwright.dev/docs/test-reporters */
+	reporter: 'html',
+	/* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+	use: {
+		/* Base URL to use in actions like `await page.goto('/')`. */
+		// baseURL: 'http://127.0.0.1:3000',
+
+		/* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+		trace: 'on-first-retry',
+		video: 'retain-on-failure',
+	},
+
+	/* Configure projects for major browsers */
+	projects: [
+		{ name: 'auth-setup', testMatch: /auth.setup\.ts/ },
+		{
+			name: 'chromium',
+			use: {
+				...devices['Desktop Chrome'],
+				storageState: 'e2e/.auth/user.json',
+			},
+			dependencies: ['auth-setup'],
+		},
+
+		// {
+		// 	name: 'firefox',
+		// 	use: { ...devices['Desktop Firefox'] },
+		// },
+		// {
+		// 	name: 'webkit',
+		// 	use: { ...devices['Desktop Safari'] },
+		// },
+		//
+		/* Test against mobile viewports. */
+		// {
+		// 	name: 'Mobile Chrome',
+		// 	use: {
+		// 		...devices['Pixel 5'],
+		// 	},
+		// },
+		// {
+		//   name: 'Mobile Safari',
+		//   use: { ...devices['iPhone 12'] },
+		// },
+
+		/* Test against branded browsers. */
+		// {
+		//   name: 'Microsoft Edge',
+		//   use: { ...devices['Desktop Edge'], channel: 'msedge' },
+		// },
+		// {
+		//   name: 'Google Chrome',
+		//   use: { ...devices['Desktop Chrome'], channel: 'chrome' },
+		// },
+	],
+
+	/* Run your local dev server before starting the tests */
+	webServer: {
+		command: 'yarn dev-app',
+		url: 'http://localhost:3000',
```

Comment:
should this go to /q to start with?

Reply:
Playwright will do a simple check if the server at this url is available. If it is then it will start the tests, otherwise it will run the command above to start the it.

Reply:
Sure sure, I understand, but I'm wondering if the initial `goto`'s might be just the tiniest bit faster if they go from the / to /q. Eh, it doesn't really matter I suppose b/c eventually it should be root / anyway when we officially launch

Reply:
Playwright won't load any website, until you tell it to. So the `goto` will be the first navigation that is performed. I think this just does a simple ping at the start, then forgets about it.

Reply:
Oh ok, thanks for clearing that up then - I didn't realize that!